ui/src/app/utils/leo-runtime-initializer.tsx

```diff
@@ -94,48 +92,8 @@ export const throwRuntimeNotFound = (
     ],
     // no current or deleted runtime exists, so we default to the general analysis runtime template
     [!currentRuntime, () => runtimePresets().generalAnalysis.runtimeTemplate],
-    // a current runtime exists and is not deleted, so we use it plus any preset overrides if appropriate
```

Comment:
All conditions below are predicated on having a deleted runtime to get information from. This is no longer a valid approach.